recipe/meta.yaml

```
@@ -25,17 +25,11 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python
-    - py4j ==0.10.9.7
-    # extras_require dependencies are kept here
-    # as runtime dependencies because previously
-    # were set as such
-    - numpy >=1.15,<2
-    - pandas >=1.0.5
-    - pyarrow >=4.0.0
-  run_constrained:
-    # We haven't previously included the "connect" extras, 
-    # so we can keep these in run_constrained.
+    - python >=3.8
+    - py4j ==0.10.9.9
+    - numpy >=1.21
+    - pandas >=2.0.0
+    - pyarrow >=11.0.0
```

```suggestion
    - python
    - py4j ==0.10.9.7
    # extras_require dependencies are kept here
    # as runtime dependencies because previously
    # were set as such
    - numpy >=1.21
    - pandas >=2.0.0
    - pyarrow >=11.0.0
  run_constrained:
    # We haven't previously included the "connect" extras, 
    # so we can keep these in run_constrained.
```
Since these were previously included as runtime requirements, they should stay that way so our users experience doesn't change.

---

Thank you for the suggestions